File: Problem2.py
def fibo():
    a = 1
    b = 2
    c = 0
    sum = 0
    while a < 4000000:
        if a % 2 == 0:
            sum += a
        c = a + b
        a = b
        b = c
    return sum


if __name__ == "__main__":
    print(fibo())

File: Problem2.rs
fn fibo() -> i128{
    let mut a:i128 = 1;
    let mut b:i128 = 2;
    let mut c:i128;
    let mut sum:i128 = 0;
    while a < 4000000{
        if a % 2 == 0{
            sum+= a;
        }
        c = a + b;
        a = b;
        b = c;
    }
    return sum;
}

fn main(){
    println!("The solution is {}",fibo());
}